splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/MultiGroupGroupedAggregateOperationIT.java

```diff
@@ -488,7 +519,159 @@ public void testRollupAllOperations() throws Exception{
         }
         Assert.assertEquals("Not all groups found!", pairStats.size()+unameStats.size()+1,row);
     }
-	
+
+    @Test
+    public void testRollupWithNotNull_1() throws Exception {
```

Comment:
What are the differences in these tests? Can those differences be reflected in the test name, rather than as _1,_2, etc?